geopandas/io/tests/test_file.py

```diff
@@ -16,6 +16,7 @@
 
 import geopandas
 from geopandas import GeoDataFrame, read_file
+from geopandas._compat import FIONA_GE_19
```

Comment:
For this file, you can define this a bit below (where there are already other version constants like this)